src/SprykerEco/Zed/Adyen/Business/Hook/Mapper/MakePayment/AbstractMapper.php

```diff
@@ -17,6 +17,8 @@
 
 abstract class AbstractMapper implements AdyenMapperInterface
 {
+    protected const GENDER_MAPPING = ['Mr' => 'MALE', 'Ms' => 'FEMALE', 'Mrs' => 'FEMALE', 'Dr' => 'MALE'];
```

Comment:
constants would be nice

Reply:
It's already in constant. I think it's okay to edit these strings here in case of changes.

Reply:
Yes, But at least MALE and FEMALE values will not change and it would be nice to have them as constants.